installer/PowerToysSetupCustomActions/CustomAction.cpp

```diff
@@ -613,9 +613,10 @@ UINT __stdcall TerminateProcessesCA(MSIHANDLE hInstall)
     }
     processes.resize(bytes / sizeof(processes[0]));
 
-    std::array<std::wstring_view, 4> processesToTerminate = {
+    std::array<std::wstring_view, 5> processesToTerminate = {
         L"PowerLauncher.exe",
         L"PowerToys.Settings.exe",
+        L"PowerToys.FancyZones.exe",
```

Comment:
Is it necessary? `PowerToys.FancyZones.exe` has to terminate if `PowerToys.exe` terminates.

Reply:
You're right, it's not necessary.